File: app/src/main/java/com/wya/env/bean/BaseResult.java
package com.wya.env.bean;

/**
 * @date: 2018/7/3 13:50
 * @author: Chunjiang Mao
 * @classname: BaseResult
 * @describe:
 */

public class BaseResult<T> {
    public int status;
    public String msg;
    public T data;
}

File: app/src/main/java/com/wya/env/bean/BaseResult.kt
package com.wya.env.bean

/**
 * @date: 2018/7/3 13:50
 * @author: Chunjiang Mao
 * @classname: BaseResult
 * @describe:
 */

data class BaseResult<T>(val status: Int,
                         val msg: String,
                         @JvmSuppressWildcards val  data: T)

